sdk/servicebus/service-bus/CHANGELOG.md

```diff
@@ -9,6 +9,12 @@
   and
   [PR 11810](https://github.com/Azure/azure-sdk-for-js/pull/11810)
 
+### Breaking changes
+
+- `ServiceBusSender` interface method `createBatch` renamed to `createMesageBatch`
+- `CreateBatchOptions` interface renamed to `CreateMessageBatchOptions`
+- `ServiceBusMessageBatch` interface method `tryAdd` renamed to `tryAddMessage`
```

Comment:
Assuming that not every user may be familiar with the interface names, consider using the general terms for the objects

```suggestion
- The `createBatch` method on the sender is renamed to `createMesageBatch`
- The interface `CreateBatchOptions` followed by the options that are passed to the `createBatch` method is renamed to `CreateMessageBatchOptions`
- The `tryAdd` method on the message batch object is renamed to `tryAddMessage`
```